content/services/design-systems-training.rst

```diff
@@ -0,0 +1,69 @@
+public: yes
+location: design-systems-training
+pageclasses: services-layout
+grid_template: true
+headline:
+  - title: 'Design Systems Training'
+    tagline: 'Want to reduce technical debt and scale your applications?'
+    type: 'Services'
+image:
+  - src: '/herman/stacy-design-systems-training.jpg'
+summary: |
+  **OddBird provides training and implementation consulting in
+  style guides, pattern libraries, layout design, and architecture.**
+  If you're a project manager, scrum master, or team leader,
+  why not make your job easier?
+
+  .. callmacro:: content.macros.j2#link_button
+    :url: '/contact/'
+
+    Let's chat about training
+
+
+Design Systems Training
+=======================
+
+.. callmacro:: content.macros.j2#rst
+  :tag: 'start'
+
+
+Your Team Will Learn How To:
+----------------------------
+
+- maintain consistentcy in the design throughout an application
+- build a foundation that leads to a scalable code base
+- discover and create meaningful patterns
+- speed up processes
+- communicate effectively as a team
+- find bugs before committing code
+- create a layout system that works for your audience
+
+
+Herman Automated Style Guide
+----------------------------
+
+`Herman`_ is a free OddBird tool that standardizes designer/developer
+communication, on-boarding, and hand-off to save time.
```

Comment:
I think it's the final "to save time" clause that still sounds odd/abrupt here to me. How about:

> Herman is a free OddBird tool that helps standardize
designer/developer communication, on-boarding, and hand-off --
saving you time and money.